### ISSUE TITLE
Design logo for Coral mission

### ISSUE THREAD

**Issue by Suzibianco** [spacedecentral/Coral#44]
Design a mission logo for Coral. It can be a logo or a mission patch, but it has to be legible even when it is very small. Please use only open source/public domain images, or your own. 
Submit a png file for approval. If you have questions about the design, please comment here.

**Comment by cagonzalez8** [comment]
I want something like this.
![av3x5po_700b](https://user-images.githubusercontent.com/42477945/46057448-39a20f80-c11b-11e8-98b5-b654a5a06950.jpg)

**Comment by Suzibianco** [comment]
Hell yeah! Hahahaha.

**Comment by cagonzalez8** [comment]
Whenever it actually happens, I'll play around with a couple main logo and a mission patch design(s) when I got some free time to kill.

**Comment by jrcgarry** [comment]
My attempt here:
(Inkscape, so all vector: and drawn while waiting for Friday rush hour to subside)
![coral crop](https://user-images.githubusercontent.com/20076169/46248309-9aa43e80-c3cc-11e8-8483-ef2ff4c5dedb.PNG)

**Comment by Suzibianco** [comment]
@jrcgarry I'm not sure what I'm looking at here, is this a cross between the moon and a coral? Nice try though!

**Comment by jrcgarry** [comment]
<sad nod>
Not exactly inspired is it?

-JG
(keeping the day job)

**Comment by cagonzalez8** [comment]
I like it. needs some color.

**Comment by jrcgarry** [comment]
I described the project to an artist - this is what she made yesterday.
<liking the blue>
![unnamed](https://user-images.githubusercontent.com/20076169/46894988-57fe4f80-ce2b-11e8-993f-f677f8dc554a.png)

**Comment by ggabogarcia** [comment]
Hello, I would like to contribute with these topics, that is why I designed this logo, for its simple forms, it works in small sizes. Any suggestions and comments are welcome.
# 
![1](https://user-images.githubusercontent.com/35750428/47619021-d273d400-daa7-11e8-9ae2-a48f1c550c5a.png)

**Comment by Suzibianco** [comment]
Hi @ggabogarcia, I'm glad you want to contribute with us. I really like your design, thank you for sharing this. I would make the lines just a bit thinner though, what do you think? And we've been kind of unofficially using purple on our publicity so far, how to you think that would work instead of the blue? I'm curious to see what the rest of the team thinks!

**Comment by ggabogarcia** [comment]
@Suzibianco  You could send me a sample of the color, or its numbering, grateful :)

**Comment by stellarmagnet** [comment]
hey @ggabogarcia I really love the direction you are taking this! Regarding the purple color -- can you try something similar to the buttons here, note that it doesn't have to be exact:
![image](https://user-images.githubusercontent.com/2584493/47668959-0fc08a80-dbaa-11e8-857b-f8706d0ce0ea.png)
![image](https://user-images.githubusercontent.com/2584493/47668972-1949f280-dbaa-11e8-984d-22001b8ab5c6.png)
- And agreed with Suzi on making the lines a little thinner. 
- How about one variation where the edges of the coral are round, instead of square?
- And a couple other font choices?

**Comment by stellarmagnet** [comment]
> I described the project to an artist - this is what she made yesterday.
> 
> ![unnamed](https://user-images.githubusercontent.com/20076169/46894988-57fe4f80-ce2b-11e8-993f-f677f8dc554a.png)

@jrcgarry i feel like this can work for one of our community update blog posts :)

**Comment by ggabogarcia** [comment]
Hello, here are the suggestions they gave me :) I think options 2 and 5 are the best.
# 
![coral](https://user-images.githubusercontent.com/35750428/47680215-38d62080-db94-11e8-96f4-77493895e9aa.jpg)

**Comment by jrcgarry** [comment]
:D
<awed>
Liking the lettering of 2 and 1.
But they're all compelling and easy on the eye.
Well done Gabriel!

**Comment by founder-outbound** [comment]
I like 2, 3, and 5. Option 3 looks like the lettering from the titles in Arthur C. Clarke’s 2001, though it may not be bold enough.

 

-Craig

**Comment by stellarmagnet** [comment]
@ggabogarcia I actually liked the black and grey you had in the previous design. Regarding the purple, the suggestion related to changing the accent color

**Comment by Suzibianco** [comment]
Yes, agreed with Yalda. The black and grey with a colored accent seemed cleaner/clearer. But I did like the difference in line width. I prefer the "sharp"corners, and of the above, my favourite was #5.

**Comment by ggabogarcia** [comment]
I made some modifications and went back to the previous color. Taking all the comments and their selections I reduced the number to two proposals. I keep leaning for the number 5 Its lines are more related to those of the isotype, besides that it tended to be better in small sizes.
# 
![2](https://user-images.githubusercontent.com/35750428/47797743-e2ccbe80-dcf4-11e8-983e-f495eec5aff0.jpg)

**Comment by ggabogarcia** [comment]
I see that this problem is not very important, anyway. Thank you very much for the opportunity :)

**Comment by stellarmagnet** [comment]
@ggabogarcia we apologize for not being more prompt with replying! We really appreciate your work on this logo.

The one comment I has is when I requested purple, I only meant the section you currently have shaded in as blue being purple. So then the logo would be purple and black, similar to your first set.

I also am leaning toward number 5!

Let us know if you can make this final change too :)

**Comment by cagonzalez8** [comment]
I'm in love with these designs @ggabogarcia!!!
